src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -1400,35 +1392,77 @@ bool GenericThreadStackManagerImpl_OpenThread<ImplClass>::IsThreadInterfaceUpNoL
     return otIp6IsEnabled(mOTInst);
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
 template <class ImplClass>
-CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::AdjustPollingInterval(void)
+CHIP_ERROR
+GenericThreadStackManagerImpl_OpenThread<ImplClass>::_GetSEDPollingConfig(ConnectivityManager::SEDPollingConfig & pollingConfig)
 {
-    CHIP_ERROR err = CHIP_NO_ERROR;
+    pollingConfig = mPollingConfig;
+    return CHIP_NO_ERROR;
+}
+
+template <class ImplClass>
+CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::_SetSEDPollingConfig(
+    const ConnectivityManager::SEDPollingConfig & pollingConfig)
+{
+    if ((pollingConfig.SlowPollingIntervalMS < pollingConfig.FastPollingIntervalMS) || (pollingConfig.SlowPollingIntervalMS == 0) ||
+        (pollingConfig.FastPollingIntervalMS == 0))
+    {
+        return CHIP_ERROR_INVALID_ARGUMENT;
+    }
+    mPollingConfig = pollingConfig;
+    return Impl()->_AdjustSEDPollingInterval(ConnectivityManager::SEDPollingIntervalType::Idle);
```

Comment:
Hmm, why do we need to reset the state to idle in that case?

Reply:
Added getting currently used polling mode